### PR TITLE
US-28 - Created card selection event/handler.

### PR DESCRIPTION
**Pull request by Pacane**
* fixed some library names

### CODE REVIEW

File: server/lib/messages/handlers/card_selection_handler.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import 'package:logging/logging.dart';
+
+import 'package:poker_planning_server/broadcaster.dart';
+import 'package:poker_planning_server/repository/game_repository.dart';
+
+import 'package:poker_planning_shared/game.dart';
+import 'package:poker_planning_shared/messages/handlers/message_handler.dart';
+import 'package:poker_planning_shared/messages/card_selection_event.dart';
+
```
Comment (christiangoudreau) [inline server/lib/messages/handlers/card_selection_handler.dart]: rm

File: shared/lib/messages/card_selection_event.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+library card_selection_event;
+
+import 'game_event.dart';
+
+class CardSelectionEvent extends GameEvent {
+  static const String MSG_TYPE = "cardSelectionEvent";
+
+  String playerName;
```
Comment (christiangoudreau) [inline shared/lib/messages/card_selection_event.dart]: How to we make that immutable in Dart?

Comment (Pacane) [inline shared/lib/messages/card_selection_event.dart]: `final` just like in java. I think you bring a good point though. The dependencies are not immutable anywhere so far. I think I'll make a task to go over every class and do this.

**Comment by meriouma** [comment]
LGTM

**Pull request merged**